pterm.go

```diff
@@ -6,20 +6,22 @@
 // View the animated examples here: https://github.com/pterm/pterm#-examples
 package pterm
 
-import "github.com/gookit/color"
+import (
+	"github.com/gookit/color"
+	"go.uber.org/atomic"
+)
 
 var (
 	// Output completely disables output from pterm if set to false. Can be used in CLI application quiet mode.
-	Output = true
+	Output = atomic.NewBool(true)
 
 	// PrintDebugMessages sets if messages printed by the DebugPrinter should be printed.
-	PrintDebugMessages = false
+	PrintDebugMessages = atomic.NewBool(false)
 
 	// RawOutput is set to true if pterm.DisableStyling() was called.
 	// The variable indicates that PTerm will not add additional styling to text.
 	// Use pterm.DisableStyling() or pterm.EnableStyling() to change this variable.
-	// Changing this variable directly, will disable or enable the output of colored text.
-	RawOutput = false
+	RawOutput = atomic.NewBool(false)
```

Comment:
Went back and forth on this part but this is the only place where the API actually changed. If I implemented with internal atomic variables like I did elsewhere it could lead to other issues. IMO these should not be exposed publicly anyways as it can lead to inconsistent usage. Looks like it may have only been done for the tests

Reply:
I think we should just expose a getter and a setter function for the atomic variables. Ubers atomic package provides much more functionality than needed. Also, if we ever decide to switch it, we would have another breaking change. A simple wrapper would fix that :)

Reply:
Agreed, will change this around. No need for a braking change here I think either.